### PR TITLE
Add strict mode info to the faq

### PR DESCRIPTION
**Pull request by fkaminski**

### CODE REVIEW

File: docs/manual/faq/_index.de.md [hunk 1]
```diff
@@ -72,6 +72,14 @@ Die Information hierzu findest du unter [TinyMCE-Editor Konfiguration](/de/anlei
 {{% /expand %}}
 
 
+## Installation
+
+{{% expand "Meldung: Ihr Datenbank-Server läuft nicht im Strict-Mode!" %}}
+Es handelt sich um einen Hinweis wie du den Strict Mode aktivieren kannst. Weitere Informationen findest du im 
+[Handbuch](https://docs.contao.org/manual/de/installation/systemvoraussetzungen/#mysql-mindestanforderungen).
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
[Handbuch](../installation/systemvoraussetzungen/#mysql-mindestanforderungen).
```

File: docs/manual/faq/_index.en.md [hunk 2]
```diff
@@ -71,6 +71,14 @@ The information about this can be found under [TinyMCE Editor Configuration](/en
 {{% /expand %}}
 
 
+## Installation
+
+{{% expand "The database server is not running in strict mode!" %}}
+This is a hint how to activate the strict mode. More information can be found in 
+the [manual](https://docs.contao.org/manual/en/installation/system-requirements/).
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.en.md]:
```suggestion
the [manual](../installation/system-requirements/).
```

File: docs/manual/faq/_index.en.md [hunk 3]
```diff
@@ -71,6 +71,14 @@ The information about this can be found under [TinyMCE Editor Configuration](/en
 {{% /expand %}}
 
 
+## Installation
+
+{{% expand "The database server is not running in strict mode!" %}}
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.en.md]:
```suggestion
{{% expand "Notification: The database server is not running in strict mode!" %}}
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

File: docs/manual/faq/_index.en.md (hunk 2 above)
Comment (fritzmg) [inline docs/manual/faq/_index.en.md]:
```suggestion
This is a hint on how to activate the strict mode. More information can be found in [here](https://docs.contao.org/manual/en/installation/system-requirements/).
```

We're already in the manual ;)

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/faq/_index.en.md (hunk 2 above)
Comment (fkaminski) [reply docs/manual/faq/_index.en.md]: Thanks - fixed that

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**